metron-platform/metron-management/README.md

```diff
@@ -276,6 +278,13 @@ The functions are split roughly into a few sections:
     * aggregatorConfig - Optional config for aggregator
   * Returns: The String representation of the enrichment config
 
+### Validation Functions
+* `VALIDATE_STELLAR_RULE_CONFIGS`
```

Comment:
This name confuses me, why Rules?  `VALIDATE_STELLAR` seems simple and to the point to me.

Reply:
Is you search the tree for 'rules' you will see that we call them rule or rules in various places.  It 'seemed like the thing to do'.

Reply:
There is reference to "rules" because that was the first use for Stellar.  @cestella called them rules when he first implemented.  But that is ancient history.  We use Stellar everywhere now.  Its long ago outgrown that.